### ISSUE TITLE
debug log may cause the data race

### ISSUE THREAD

**Issue by detailyang** [copernet/copernicus#42]
Hello.
I'm not sure it should be fixed or not. The debug log may cause the data race as the following or [syncmanager.go#L1078](https://github.com/copernet/copernicus/blob/master/net/syncmanager/syncmanager.go#L1078)

````go
	log.Debug(
		"invBlkCnt=%d len(invVects)=%d sm.requestBlkInv=%v  peer=%p(%#v) sm.syncPeer=%p",
		invBlkCnt, len(invVects), sm.requestBlkInvCnt, peer, peer, sm.syncPeer)
````

The verbs `%#v` will case the peer's fields data race because of the reflect
 after receiving inv message and you can easily reproduce it when build with `go build -race`

**Comment by wolfstudy** [comment]
@qiwei9743 PTAL

**Comment by qiwei9743** [comment]
Good catch. I'll fix it later. Thank you.

**Comment by qiwei9743** [comment]
fixed by commit a75ab2f.
